Sift/Sift.csproj

```diff
@@ -1,11 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What is changed at this line?

Reply:
@advornichenko-sift there doesn't seem to be any change, not sure why GitHub highlighted this line

Reply:
actually it looks like encoding changed from UTF-8 BOM to UTF-8, I'll try to revert it